.neuro/hypertrain.yml

```diff
@@ -0,0 +1,64 @@
+kind: batch
+
+params:
+  token:
+    descr: W&B auth token
+
+volumes:
+  data:
+    remote: storage:$[[ flow.project_id ]]/data
+    mount: /project/data
+  code:
+    remote: storage:$[[ flow.project_id ]]/src
+    mount: /project/code
+  config:
+    remote: storage:$[[ flow.project_id ]]/config
+    mount: /project/config
+    read_only: True
+  notebooks:
+    remote: storage:$[[ flow.project_id ]]/notebooks
+    mount: /project/notebooks
+  results:
+    remote: storage:$[[ flow.project_id ]]/results
+    mount: /project/results
+  project:
+    remote: storage:$[[ flow.project_id ]]
+    mount: /project
+
+images:
+  myimage:
+    ref: image:$[[ flow.project_id ]]
+    dockerfile: $[[ flow.workspace ]]/Dockerfile
+    context: $[[ flow.workspace ]]/
+    build_preset: cpu-small   
+
+tasks:
+  - id: start_sweep
+    image: $[[ images.myimage.ref ]]
+    preset: cpu-small
+    volumes:
+      - $[[ volumes.config.ref ]]
+      - $[[ volumes.project.ref ]]
+    bash: |
+      wandb login $[[ params.token ]]
+      WANDB_SWEEPS_FILE="$[[ volumes.config.mount ]]/wandb-sweep.yaml"
+      SWEEP=$(WANDB_PROJECT=ml-recipe-hyper-search wandb sweep $WANDB_SWEEPS_FILE 2>&1 | grep 'sweep with ID:' | cut -d' ' -f6)
+      echo ::set-output name=sweep::${SWEEP}
+
+  - id: worker_$[[ matrix.id ]]
+    image: $[[ images.myimage.ref ]]
+    preset: gpu-k80-small-p
+    needs: [start_sweep]
+    strategy:
+      matrix:
+        id: [1, 2]
```

Comment:
Is it possible in principle to parameterize the matrix values? (haven't checked it yet, but curious)
If yes, It would be cool to expose one more arg to this pipeline, say, "number_of_workers".
And set the matrix ids in start_sweep task via task output.

Reply:
Sadly, it's not supported now. I discussed this with Roman and he suggested us to implement a `range(N)` function for neuro-flow, which we might have some time for in the future (doesn't seem to be a priority now as the recipe is not really used by anyone lol)

Reply:
> doesn't seem to be a priority now as the recipe is not really used by anyone lol

Sooo true 🤣 
Ok, got you, thanks for the explanation!